src/main/java/io/vertx/core/net/TCPSSLOptions.java

```diff
@@ -405,6 +410,50 @@ public TCPSSLOptions addCrlPath(String crlPath) throws NullPointerException {
   }
 
   /**
+   * Add key options for SNI host.
+   *
+   * @param domain The domain name to match. May contain wildcards.
+   * @param keyCertOptions The key options to use when matching this host.
+   * @return a reference to this, so the API can be used fluently
+   */
+  public TCPSSLOptions addSNIKeyCertOptionsForDomain(String domain, KeyCertOptions keyCertOptions) throws NullPointerException {
```

Comment:
there is notion of "adder" where getter + adder match together in data object:

public Map<String, String> getValues() { ... }
public void addValue(String key, String value) { ... }

notice the "s" difference.

so we should have this correspondance in the data object